modules/iform_licences/iform_licences.module

```diff
@@ -68,13 +67,22 @@ function iform_licences_form_alter(&$form, &$form_state, $form_id) {
             );
           }
           else {
-            $description .= strtoupper(
+            $description .= '<b>' . strtoupper(
               t('This licence will also be applied to all the @label you have previously uploaded to @site.',
               [
                 '@label' => $label,
                 '@site' => variable_get('site_name', 'this system'),
               ]
-            ));
+            )) . '</b>';
+            if ($licenceFor === 'media') {
```

Comment:
Does this need to account for records+media mode?

Reply:
@johnvanbreda if configured for media & records, media and records are entered as two separate elements in array and code cycles through each, giving two separate option blocks on form.

![image](https://user-images.githubusercontent.com/9275178/89404629-3a8e2a00-d712-11ea-80aa-9a0d1cc43e20.png)

Reply:
Just to confirm that we do **_not_** wish to implement record-level licensing at the moment.

Reply:
That's fine @kitenetter - just need to make sure it all works for situations where both are configured.